tests/back_compat/test_import_structure.py

```diff
@@ -0,0 +1,84 @@
+from unittest import TestCase
+
+import pynwb
+
+
+class TestImportStructure(TestCase):
+    def test_outer_import_structure(self):
```

Comment:
Could you add a docstring to clarify what the purpose of this test function/class is.

Reply:
Done

Reply:
I thought the same thing